### PR TITLE
added styles to the order success page

### PR DESCRIPTION
**Pull request by cjdeg**

### CODE REVIEW

File: src/components/store/pages/callbacks/success.js [hunk 1]
```diff
@@ -19,14 +19,74 @@ export default () => {
...
   return (
-    <Page title="Success">
-      <div className="column">
-        {!checkoutStatus && (
-          <p>Please wait while we load your order: {creditKeyId}</p>
-        )}
-        {checkoutStatus && (
-          <p>Your order with id "{creditKeyId} has been confirmed. Please wait for the hampsters to generate enough power for delivery. Good Day."</p>
-        )}
+    <Page title="">
```
Comment (johnTurknett) [inline src/components/store/pages/callbacks/success.js:22]: Do we need a title here?

File: src/components/store/pages/callbacks/success.js [hunk 2]
```diff
@@ -19,14 +19,74 @@ export default () => {
...
+      <div className="columns">
+
+        <div className="column is-7">
+          <div className="thank-you">
+            
+              <img className="blue-checkmark" src="/images/blue_checkmark.png" alt="Blue Checkmark"/>
+            
+            <div className="thank-you-text">
+              <p>Order #{creditKeyId}</p>
+              <div className="title">Thanks you!</div>
```
Comment (johnTurknett) [inline src/components/store/pages/callbacks/success.js:32]: spelling/Thanks/Thank

**Review by johnTurknett — CHANGES_REQUESTED** [review]
Just some small things noticed.

**Comment by cjdeg** [comment]
OK @johnTurknett - should be all set. Thanks

**Review by johnTurknett — APPROVED** [review]

**Pull request merged**